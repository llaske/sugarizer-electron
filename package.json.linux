{
  "name": "Sugarizer",
  "version": "1.4.0",
  "main": "main.js",
  "scripts": {
    "start": "electron .",
    "dist": "electron-builder --linux --x64 --ia32"
  },
  "repository": "https://github.com/llaske/sugarizer",
  "author": "Lionel Laské <lionel@sugarizer.org>",
  "description": "A taste of Sugar on any device",
  "license": "Apache-2.0",
  "dependencies": {
    "requirejs": "~2.3.5",
    "tmp": "~0.1.0",
    "ini": "1.1.x"
  },
  "devDependencies": {
    "grunt": "~1.2.1",
    "grunt-contrib-jshint": "~2.1.0",
    "grunt-contrib-nodeunit": "~2.1.0",
    "grunt-terser": "~1.0.0",
    "electron": "~7.2.0"
  },
  "build": {
    "appId": "org.olpc-france.sugarizer",
    "copyright": "Copyright © 2013-2020, Lionel Laské, Sugar Labs Inc. and Contributors",
    "productName": "Sugarizer",
    "electronVersion": "7.2.4",
    "mac": {
      "category": "public.app-category.education"
    },
    "win": {
      "target": [ "nsis" ]
    },
    "linux": {
      "target": [ "AppImage", "deb" ]
    },
    "snap": {
      "plugs": ["desktop", "desktop-legacy", "home", "x11", "unity7", "network", "gsettings", "pulseaudio", "opengl", "camera", "browser-support"]
    }
  }
}
