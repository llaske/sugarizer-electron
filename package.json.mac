{
  "name": "Sugarizer",
  "version": "1.7.0",
  "main": "main.js",
  "scripts": {
    "start": "electron .",
    "dist": "electron-builder -m"
  },
  "repository": "https://github.com/llaske/sugarizer",
  "author": "Lionel Laské <lionel@sugarizer.org>",
  "description": "The leading learning platform for children",
  "license": "Apache-2.0",
  "dependencies": {
    "requirejs": "~2.3.5",
    "tmp": "~0.1.0",
    "ini": "~1.3.6"
  },
  "devDependencies": {
    "grunt": "~1.6.0",
    "terser": "~5.16.0",
    "grunt-terser": "~2.0.0",
    "grunt-contrib-jshint": "~2.1.0",
    "grunt-contrib-nodeunit": "~2.1.0",
    "electron": "~22.3.1"
  },
  "build": {
    "appId": "org.olpc-france.sugarizer",
    "copyright": "Copyright © 2013-2023, Lionel Laské, Sugar Labs Inc. and Contributors",
    "productName": "Sugarizer",
    "electronVersion": "22.3.1",
    "mac": {
      "category": "public.app-category.education"
    },
    "win": {
      "target": [
        "nsis"
      ]
    }
  }
}
