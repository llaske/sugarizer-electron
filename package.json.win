{
  "name": "Sugarizer",
  "version": "1.9.0",
  "main": "main.js",
  "scripts": {
    "start": "electron .",
    "dist": "electron-builder -w"
  },
  "repository": "https://github.com/llaske/sugarizer",
  "author": "Lionel Laské <lionel@sugarizer.org>",
  "description": "The leading learning platform for children",
  "license": "Apache-2.0",
  "dependencies": {
    "requirejs": "~2.3.5",
    "normalize-url": "~4.5.1",
    "hosted-git-info": "~2.8.9",
    "tmp": "~0.1.0",
    "axios": "^0.27.2",
    "i18next": "~20.3.1",
    "ini": "~1.3.6"
  },
  "devDependencies": {
    "grunt": "~1.6.0",
    "terser": "~5.16.0",
    "grunt-terser": "~2.0.0",
    "grunt-contrib-jshint": "~2.1.0",
    "grunt-contrib-nodeunit": "~2.1.0",
    "electron": ">=22.3.25"
  },
  "build": {
    "appId": "org.olpc-france.sugarizer",
    "copyright": "Copyright © 2013-2025, Lionel Laské, Sugar Labs Inc. and Contributors",
    "productName": "Sugarizer",
    "electronVersion": "22.3.25",
    "mac": {
      "category": "public.app-category.education"
    },
    "win": {
      "target": [
        "nsis"
      ]
    }
  }
}
