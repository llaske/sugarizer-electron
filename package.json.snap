{
  "name": "Sugarizer",
  "version": "1.3.0",
  "main": "main.js",
  "scripts": {
    "start": "electron .",
    "dist": "electron-builder --linux snap"
  },
  "repository": "https://github.com/llaske/sugarizer",
  "author": "Lionel Laské <lionel@sugarizer.org>",
  "description": "A taste of Sugar on any device",
  "license": "Apache-2.0",
  "dependencies": {
    "requirejs": "~2.3.5",
    "tmp": "~0.1.0",
    "ini": "1.1.x"
  },
  "devDependencies": {
    "electron": "~6.0.0",
    "grunt": "~0.4.5",
    "grunt-contrib-jshint": "~0.10.0",
    "grunt-contrib-nodeunit": "~0.4.1",
    "grunt-contrib-uglify": "~0.5.1"
  },
  "build": {
    "appId": "org.olpc-france.sugarizer",
    "copyright": "Copyright © 2013-2020, Lionel Laské, Sugar Labs Inc. and Contributors",
    "productName": "Sugarizer",
    "electronVersion": "6.0.9",
    "mac": {
      "category": "public.app-category.education"
    },
    "win": {
      "target": [ "nsis" ]
    },
    "linux": {
      "target": [ "snap" ]
    },
    "snap": {
      "plugs": ["desktop", "desktop-legacy", "home", "x11", "unity7", "network", "gsettings", "pulseaudio", "opengl", "camera", "browser-support"]
    }
  }
}
